ipfs-cluster-service/main.go

```diff
@@ -145,9 +148,9 @@ func checkErr(doing string, err error, args ...interface{}) {
 }
 
 func main() {
-	// go func() {
-	//	log.Println(http.ListenAndServe("localhost:6060", nil))
-	// }()
+	go func() {
+		log.Println(http.ListenAndServe("localhost:6060", nil))
```

Comment:
Let's put the listen options in a variable. And let's enable this endpoint only with a metrics flag? it only makes sense when running `daemon`. So something like `ipfs-cluster-service daemon --metrics`?